cli/lib/kontena/cli/stacks/deploy_command.rb

```diff
@@ -21,7 +23,9 @@ def execute
       spinner "Waiting for deployment to start" do
         wait_for_deployment_to_start(deployment)
       end
-      wait_for_deploy_to_finish(deployment)
+      if !async?
```

Comment:
I think we're using more `unless self.async?` style for these kind of checks, reads a bit better :)